### PR TITLE
FHIR Condition Resource

### LINKED ISSUE #3346: Addition of FHIR Condition and Immunization Resources
### Describe the problem
Lets track these resources here. @silvistershebin whichever one you want. Just note it here.

### PR DESCRIPTION
**Pull request by silvistershebin**
Fixes #3346 

#### Short description of what this resolves:
FHIR Condition Resource with GET methods

### CODE REVIEW

File: _rest_routes.inc.php [hunk 1]
```diff
@@ -404,5 +404,13 @@
     },
     "GET /fhir/Immunization/:id" => function ($id) {
         return (new FhirImmunizationRestController(null))->getOne($id);
+    },
+    "GET /fhir/Condition" => function () {
+        RestConfig::authorization_check("patients", "med");
```
Comment (bradymiller) [inline _rest_routes.inc.php:409]: for the 2 immunization routes above this, they should also have this same auth check
```
RestConfig::authorization_check("patients", "med");
```

Comment (silvistershebin) [reply _rest_routes.inc.php:409]: Ok I'll fix those

**Comment by silvistershebin** [comment]
Yes, I'll push the updates soon.

**Comment by silvistershebin** [comment]
Not sure what's on with travis here :(

**Comment by sjpadgett** [comment]
I'm not worried about build errors. PSR is okay.
Thanks for the contribution, @silvistershebin

**Comment by bradymiller** [comment]
yes, travis is acting out. I think travis is displaying passive/aggresive behavior towards @sjpadgett . will need to officially ignore travis; hopefully this does not upset travis :fearful: 

tenor (5)

**Pull request merged**